package.json

```diff
@@ -4,8 +4,8 @@
   "license": "W3C",
   "description": "A technical specification pre-processor.",
   "engines": {
-    "node": ">=10",
-    "npm": ">=5"
+    "node": ">=14",
```

Comment:
I don't think the code would have run on 10... so this is a bug  fix.... but!!!!